### ISSUE TITLE
Order Healbless stacking vs Destro not stacking

### ISSUE THREAD

**Issue by Shadowgurke** [WarEmu/WarBugs#9284]
Hey friends,
Order Blessing of Grungi + Magical Infusion from RP and AM stack additively.

Destro Blessing of Chaos + Shrug it off from Zealot and Shaman don't stack at all.

If that is intentional then ignore this post.

![destrohb](https://cloud.githubusercontent.com/assets/1018113/23634317/e36df87e-02c9-11e7-9671-be9279502bd6.jpg)
![orderhb](https://cloud.githubusercontent.com/assets/1018113/23634320/e611d776-02c9-11e7-9cbd-637ee7468926.jpg)

**Comment by Luth-RoR** [comment]
>Magical Infusion and Shrug It Off will increase the value of heals that a target receives by 25% when active.

https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=17985

**Comment by Shadowgurke** [comment]
I know. Thy both mimic the zealot/rp buffs, However, Magical Infusion stacks with RP blessing.
410 Base
410+25% Blessing of Grungi(102.5)=513
410+25% Blessings +25% Magical Infusion = 615

On Destro side, they don't stack. Both give the 25% buff but having both on the target is no different than having one of them.

**Comment by Luth-RoR** [comment]
Yes i understood; it was just meant as a confirmation that it should work either on both sides or none, not only one.

**Comment by argott** [comment]
i am sorry for cutting in but i just checked and issue is still there. is there any way to solve it?

**Comment by Zomega-WAR** [comment]
This is now fixed and can be closed. Test below. The combat log was a bit noisy with crits/etc so I've put the 3 relevant lines here. This was tested with a Shaman using "Ain't Done Yet" (the tactic that improves the incoming healing of a target affected by Shrug It Off) and the Zealot's Blessing of Chaos.

Elixir of Dark Blessings: 1228 (no buffs)
Elixir of Dark Blessings: 1535 (one buff, 25% increase)
Elixir of Dark Blessings: 1842 (two buffs, 50% increase)

![image](https://user-images.githubusercontent.com/60266177/121791388-44f63b80-cbe1-11eb-84cf-0929888fbbdc.png)
